src/pieces/logs/serverLog.ts

```diff
@@ -16,7 +16,8 @@ import {
 	ThreadChannel
 } from 'discord.js';
 import prettyMilliseconds from 'pretty-ms';
-import { GUILDS, CHANNELS } from '@root/config';
+import { GUILDS, CHANNELS, DB } from '@root/config';
+import { SageUser } from '@root/src/lib/types/SageUser';
```

Comment:
You should utilize `@lib`
```suggestion
import { SageUser } from '@lib/types/SageUser';
```